.github/workflows/main.yml

```diff
@@ -5,7 +5,11 @@ on:
     branches:
       - master
       - develop
-  pull_request: ~
+  pull_request:
+    paths-ignore:
+      - '**/*.md'
+      - '**/package.json'
```

Comment:
It doesn't look like the lockfile changes from the [consume changesets PRs](https://github.com/smartcontractkit/external-adapters-js/pull/2120/files), which in a way protects us, because dependency changes will still run the main workflow.